android/assets/jsons/Civ V - Vanilla/Policies.json

```diff
@@ -76,6 +76,7 @@
 			{
 				"name": "Meritocracy",
 				"uniques": ["[+1 Happiness] [in all cities connected to capital]", "Unhappiness from population decreased by [5]%"],
+				// Should only be unhpainness from population in non-occupied cities
```

Comment:
µBlock filter `##form.js-inline-comment-form  button.btn-primary` works... Anyway, 'unhappiness', no pain. But - the comments in json are illegal json issue is still kind of hanging and Android Studio will complain, I'd not create new json comments unless painfully necessary.

Reply:
While I understand your reasoning, I don't particularly mind adding comments myself. In the cases where I add it, I do it as an indication that there is still work to be done, and if completed, the comment should of course be removed. This way, in the 'finished product', the comments would be gone, but while developing they are useful in this sense and since our parser is able to handle them, I don't feel like there is a big downside to them.